packages/console/src/ds-components/Select/index.module.scss

```diff
@@ -16,6 +16,40 @@
   cursor: pointer;
   position: relative;
 
+  &.multiple {
+    justify-content: flex-start;
+    flex-wrap: wrap;
+    gap: _.unit(2);
+    padding: _.unit(2) _.unit(3);
+    cursor: text;
+
+    .tag {
+      cursor: auto;
+      display: flex;
+      align-items: center;
+      gap: _.unit(1);
+    }
+
+    .close {
+      width: 16px;
+      height: 16px;
+    }
```

Comment:
`cursor: pointer?`
Or maybe we can make the entire `.tag` show the pointer cursor on mouse hover

Reply:
it's a button. already pointer

Reply:
nah, tag is tag